modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/display/ManageUserAssociatedDataEntitiesDisplay.java

```diff
@@ -39,7 +43,30 @@ public UADEntityDisplay getUADEntityDisplay() {
 		return _uadEntitySearchContainer;
 	}
 
+	public String getUADEntitySetName() {
+		return _uadEntitySetName;
+	}
+
+	public String getUADRegistryKey() {
+		return _uadRegistryKey;
+	}
+
+	public void setCount(int count) {
+		_count = count;
+	}
+
+	public void setUAdEntitySetName(String uadEntitySetName) {
+		_uadEntitySetName = uadEntitySetName;
+	}
+
+	public void setUAdRegistryKey(String uadRegistryKey) {
+		_uadRegistryKey = uadRegistryKey;
+	}
+
+	private int _count;
```

Comment:
The search container will already have the count in it, so we shouldn't need an extra field here.